model_compression_toolkit/core/keras/back2framework/mixed_precision_model_builder.py

```diff
@@ -22,10 +22,11 @@
     from keras.engine.base_layer import Layer
 
 from keras.models import Model
-from mct_quantizers import KerasQuantizationWrapper, KerasActivationQuantizationHolder, QuantizationTarget
```

Comment:
I think that mixed precision model doesn't need to use KerasTrainableQuantizationWrapper but the original KerasQuantizationWrapper (verify this)